bucket/lib.rs

```diff
@@ -314,14 +314,35 @@ pub mod ddc_bucket {
     }
 
     impl DdcBucket {
-        /// Create a new cluster and return its `cluster_id`.
-        ///
-        /// The caller will be its first manager.
+        /// Description:
+        ///     Create a new cluster.
         ///
-        /// The cluster is split in a number of vnodes. The vnodes are assigned to the given physical nodes in a round-robin. Only nodes of providers that trust the cluster manager can be used (see `node_trust_manager`). The assignment can be changed with the function `cluster_replace_node`.
+        /// Logic:
+        ///     The caller will be its first manager.
+        ///     The cluster is split in a number of vnodes. The vnodes are assigned to the given physical nodes in a round-robin. 
+        /// 
+        /// Permissions:
+        ///     Only nodes of providers that trust the cluster manager can be used (see `node_trust_manager`). The assignment can be changed with the function `cluster_replace_node`.
         ///
-        /// `cluster_params` is configuration used by clients and nodes. In particular, this describes the semantics of vnodes. See the [data structure of ClusterParams](https://docs.cere.network/ddc/protocols/contract-params-schema)
-        #[ink(message, payable)]
+        /// Params:
+        ///     `cluster_params` is configuration used by clients and nodes. In particular, this describes the semantics of vnodes. See the [data structure of ClusterParams](https://docs.cere.network/ddc/protocols/contract-params-schema)
+        /// 
+        /// Returns:
+        ///     `cluster_id` - id of the newly created cluster.
+        /// 
+        /// Events:
+        ///     ClusterCreated event is triggered, which includes clusterid, managerid, and cluster params
```

Comment:
Here can be few events.
Will we use some list? Will we use the list even if we have only one item?

Reply:
updated